### PR TITLE
comment

### PR DESCRIPTION
**Pull request by Michael-Mc-Mahon**
This is my first pull request

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * Chris Hegarty ([chegar](@ChrisHegarty) - **Reviewer**) ⚠️ Review applies to ec441314867274bd77726aa50f4f1605b84be263

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/27/head:pull/27`
`$ git checkout pull/27`

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+Welcome to the JDK!
```
Comment (ChrisHegarty) [inline README:2]: What did you change here?

**Review by ChrisHegarty — CHANGES_REQUESTED** [review]

File: README (hunk 1 above)
Comment (Michael-Mc-Mahon) [reply README:2]: The top level README in open

Comment (AlanBateman) [reply README:2]: I was wondering the same thing

**Review by ChrisHegarty — APPROVED** [review]
LGTM.

**Comment by dfuch** [comment]
Why are the bots /help answer visible in this chat?

**Pull request closed without merging**